ClientSideKeyVaultKeyToCustomerManagedKey/config.py

```diff
@@ -0,0 +1,24 @@
+from azure.keyvault.keys.crypto import KeyWrapAlgorithm
+# the below values are fake and need to be replaced
+
+KEYVAULT_URL = ""
+CLIENT_ID = ""
+CLIENT_SECRET = ""
+TENANT_ID = ""
+connection_str = ""
+
+# items to make encryption scope
+SUB_ID = ""
+RESOURCE_GROUP = ""
+STORAGE_ACCOUNT = ""
+
+# replace with name for customer managed encryption scope
+customer_managed_encryption_scope = "test-customer-scope"
+keyname = "testkey1"
+serverside_encryption_keyname = "testkey2"
+secret = "sample-secret"
```

Comment:
what are these two?

Reply:
keyname vs serverside keyname was so that the user can use a different key for serverside encryption than what they used for client side encryption with keyvault

Reply:
it looks like serverside_encryption_keyname is never used in miigration file

---
In reply to: [466777955](https://github.com/wastore/azure-storage-samples-for-python/pull/5#discussion_r466777955) [](ancestors = 466777955)

Reply:
yeah! it's actually used in the sampleDataCreation file to make the encryption scope. So, the keyname key that the user used for client side encryption is different from the encryption scope, made from the serverside_encryption_keyname key, to perform serverside encryption